src/components/search/search.njk

```diff
@@ -1,35 +1,35 @@
 <h6>Search big</h6>
 
-<div role="search">
-  <form class="usa-search usa-search-big">
+<form class="usa-search usa-search-big">
```

Comment:
would it make sense to include the `role` with the `form` element, remove the inner `div`, and consolidate the styles?

Reply:
Are we gaining something important by keeping them separate?

Reply:
Yeah for a11y reasons http://www.a11ymatters.com/pattern/accessible-search/